build.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
     id("com.github.jakemarsden.git-hooks")
     id("com.github.johnrengelman.shadow")
     id("io.gitlab.arturbosch.detekt")
+    id("com.expediagroup.graphql") version "5.2.0"
```

Comment:
Specify the version in `settings.gradle.kts` as all the other plugins have been.

Reply:
You're still missing this change, otherwise we're good